src/js/letters/containers/AddressSection.jsx

```diff
@@ -45,6 +50,8 @@ export class AddressSection extends React.Component {
     // savedAddress values
     if (Object.keys(this.state.editableAddress).length > 0) {
       this.state.hasLoadedAddress = true;
+      // If we start with an empty address, go straight to editing
+      this.state.isEditingAddress = isAddressEmpty(this.state.editableAddress);
```

Comment:
Don't think this is a huge deal, but I do get some UI thrashing with this approach - at first I see the 'Address Update Not Available' warning, then the editor. Takes a second or two to switch over. This is probably because the full states and countries list isn't available right away (Address Section L298), but we probably don't need to care about that since this is already an edge case.

Reply:
I don't think that's because of this, actually. We'll get that warning when `this.props.addressAvailability === AVAILABILITY_STATUSES.unavailable`, so it's just the lag between when this component mounts and when we get the response back from the api to set `addressAvailability` to `available`.

We should probably write a ticket for that, but it's not anything new here.